_dev/js/front/src/service/prestashop.service/prestashop-ps1_7.service.js

```diff
@@ -67,6 +75,14 @@ export class PrestashopPs1_7Service {
     return !!window.ps_checkoutCartProductCount;
   }
 
+  static getCartAmount() {
+    return window.prestashop.cart.totals.total.amount || '';
+  }
+
+  static getProductPrice() {
+    return document.querySelector('.current-price .current-price-value').getAttribute('content') || '';
```

Comment:
This code doesn't work. On classic theme this works:
    return document.querySelector('.current-price [itemprop="price"]').getAttribute('content') || '';

Reply:
@L3RAZ It works for me on 1.7.8.5 and classic theme, on which version did you test ? Maybe I need to test different versions and custom themes

Reply:
@Matt75 I was testing on 1.7.7.1 classic theme